src/WooCommerce.php

```diff
@@ -1492,4 +1492,34 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
     return $dropdown_values;
   }
 
+  /**
+   * Adds custom meta to customer.
+   *
+   * @implements woocommerce_checkout_create_order
+   */
+  public static function addCustomMetaForUser($order) {
+    $user_id = $order->get_user_id();
+    $items_counter = $order->get_item_count();
+    $customer_orders_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_orders_count', TRUE);
+    $customer_order_items_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_order_items_count', TRUE);
+    if ($customer_orders_count && $customer_order_items_count) {
+      update_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_orders_count', $customer_orders_count + 1);
+      update_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_order_items_count', $customer_order_items_count + $items_counter);
```

Comment:
```suggestion
      update_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_orders_count', $customer_orders_count + 1);
      update_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_order_items_count', $customer_order_items_count + $items_counter);
```

I see this exact code running twice.  Can we DRY it?

Reply:
Updated in the last commit. Can you tell me please if this is a good approach?

Reply:
I updated this in the last commit. Is this a good approach for DRY?

Reply:
Yes! Thanks.